tests/integrations/conftest.py

```diff
@@ -85,7 +85,7 @@ async def _factory(name, *args, **kwargs):
         setattr(kakarot_contract, "contract_account", contract_account)
         deployed_contracts[name] = kakarot_contract
 
-        return kakarot_contract
+        return kakarot_contract, tx
```

Comment:
Adding `tx` here to get the resulting `evm_contract_address` and `starknet_contract_address` after contract is deployed thru kakarot

Reply:
this was my first implementation but really disliked eventually this returned tuple, that's why I added an attribute "contract_account".
You should be able to retrieve it using `kakarot_contract.contract_account.deploy_call_info`

Reply:
Ohh thanks @ClementWalter ! I missed this nice attribute from the wrapper you made, awesome 🙂 I will use that instead 👌